### ISSUE TITLE
Virtual keyboard hides items and screen scroll does not occur

### ISSUE THREAD

**Issue by evertonfoz** [salim-lachdhaf/searchable_dropdown#217]
When focus goes to the search box, the keyboard hides the items. tried using the component in a listview and forcing a jump, but it doesn't work either. Can you help?

![image](https://user-images.githubusercontent.com/9963789/126872402-d0bab1d8-0a35-4c2c-84cb-201370ac9fbd.png)

**Comment by vasilich6107** [comment]
Clod you read this https://github.com/salim-lachdhaf/searchable_dropdown/issues/150#issuecomment-840082887

**Comment by evertonfoz** [comment]
Okay. I changed it to bottomsheet, and the rounded edges appear with the background color. Shouldn't they be transparent?

```
popupShape: RoundedRectangleBorder(
        // gapPadding: 100,
        borderRadius: BorderRadius.only(
          topLeft: Radius.circular(20.0),
          topRight: Radius.circular(20.0),
        ),
        // gapPadding: 50,
        side: BorderSide(
          color: Colors.red.shade900,
          width: 2.0,
          // style: BorderStyle.solid,
        ),
      ),
```

![image](https://user-images.githubusercontent.com/9963789/126875649-6aa5e74f-90e1-4ee7-bded-ddc7a5e1a451.png)

**Comment by vasilich6107** [comment]
Will check later this week

**Comment by vasilich6107** [comment]
@salim-lachdhaf I also added `popupElevation` property in https://github.com/artflutter/searchable_dropdown/tree/elevation-property

Which should be merged after #220
